bot/cogs/free.py

```diff
@@ -72,30 +72,32 @@ class Free(Cog):
         # Display all potentially inactive channels
         # in descending order of inactivity
         if free_channels:
-            embed.description += "**The following channel{0} look{1} free:**\n\n**".format(
-                's' if len(free_channels) > 1 else '',
-                '' if len(free_channels) > 1 else 's'
-            )
-
             # Sort channels in descending order by seconds
             # Get position in list, inactivity, and channel object
             # For each channel, add to embed.description
             sorted_channels = sorted(free_channels, key=itemgetter(0), reverse=True)
+            if len(sorted_channels) > 3:  # display 3 channels max
+                sorted_channels = sorted_channels[:3]
+
             for i, (inactive, channel) in enumerate(sorted_channels, 1):
                 minutes, seconds = divmod(inactive, 60)
```

Comment:
Just because I got a similar review comment, could this utilise the `utils.time.humanize_time` method, if you cast `inactive` to a timedelta?

Reply:
Don't think so because it uses full names for units instead of single letters.

However, looking over the time-related code here, it could be improved. The delta object could be appended instead of just the seconds to avoid having to do the redundant math below to get the minutes and hours.

Reply:
To be clear, I acknowledge this is out of scope of the issue you are addressing but it would be a welcome addition if you wish to do it @kraktus.

Reply:
I am sorry but I didn't quite understand how to simplify time-related, because time delta object does not have a hour attribute.

Reply:
Oh, that's annoying. Well never mind then.